src/main/java/frc/robot/hardware/signal/AngleSignal.java

```diff
@@ -43,6 +44,26 @@ public double[] getTimestamps() {
 		return new double[] {timestamp};
 	}
 
+	@Override
+	public boolean isNear(Rotation2d value, Rotation2d tolerance) {
+		return MathUtil.isNear(value.getRotations(), getLatestValue().getRotations(), tolerance.getRotations());
+	}
+
+	@Override
+	public boolean isFarBy(Rotation2d distance) {
+		return !isNear(distance, Rotation2d.fromRotations(0));
+	}
```

Comment:
this is not how math works.
also u want to get value and return if the distnace of current to given value is bigger that given distnace